openstack/octavia/templates/octavia-api-ingress.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
     component: octavia
 {{- with .Values.ingress.annotations }}
   annotations:
+  {{- include "utils.linkerd.pod_and_service_annotation" $ | indent 4 }}
 {{ toYaml . | indent 4 }}
 {{- end }}
```

Comment:
This needs to change, because when `.Values.ingress.annotations` is empty, we will not add the `annotations` key and the linkerd annotation. While `.Values.ingress.annotations` is currently not empty by default, the logic is still flawed.

Additionally, this needs to be `utils.linkerd.ingress_annotation`.

Reply:
I fixed wrong annotation and changed order of things, annotation key is not any more subject to `.Values.ingress.annotations`